snakefile

```diff
@@ -99,6 +99,62 @@ rule extract_geoboundaries:
     shell:
         "extract_geoboundaries"
 
+## OSM Extract ##
```

Comment:
appears to be replacing _extract_roads_osm_ with _extract_osm_roads_ in the pull request. This should be fine. Just need to double check old rules are being replaced